expected_output:
  - The pod `db-certs-authenticator` in the `default` namespace is experiencing a `CrashLoopBackOff`
  - Mention the runbook `KubePodCrashLooping`
before_test: |
  kubectl apply -f https://raw.githubusercontent.com/robusta-dev/kubernetes-demos/main/crashpod.v2/crashloop-cert-app.yaml
  sleep 120
after_test: |
  kubectl delete -f https://raw.githubusercontent.com/robusta-dev/kubernetes-demos/main/crashpod.v2/crashloop-cert-app.yaml

evaluation:
  correctness: 1
